Global/Emacs.gitignore

```diff
@@ -7,6 +7,13 @@
 auto-save-list
 tramp
 .\#*
+/games/
```

Comment:
It's not clear to me where these rules are coming from, and what they represent. Could you elaborate?